README.md

```diff
@@ -1,2 +1,104 @@
-# pacta.sit.rep
-A simple situation report of the overall code health of the PACTA ecosystem. 
+# pacta.sit.rep <img src="assets/images/logo.png" align="right" width="120" />
+<!-- badges: start -->
+
+[![Lifecycle: experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html)
+<!-- badges: end -->
+
+This repository generates a simple situation report of the overall code health of the PACTA ecosystem.
+
+Code health is reported at the level of the R package, and at the level of the Transition Monitor Docker image. 
+
+## R Packages
+| Product            	| Repo                                                                                            	| Lifecycle                                                                                                                                     	| Status                                                                                                                                                                                                                       	| latest `main`                                                                 	| Maintainer                              	|
```

Comment:
It seems like all the badges are squished to fit in the markdown table.

Normally, I wouldn't make that a blocking comment, but here it seems like it should be.

Reply:
Ok, I split out the tables, so that the "PRODUCT" is now a markdown heading. And i also removed an excessively long comment, so the badges seem to render better.